code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -3412,6 +3430,33 @@
 /datum/reagent/methylin/on_overdose(var/mob/living/M)
 	M.adjustToxLoss(1)
 	M.adjustBrainLoss(1)
+	
+/datum/reagent/mannitol
+	name = "Mannitol"
+	id = MANNITOL
+	description = "Mannitol is strange yet commonly requested mixture that is said to quell headaches."
+	reagent_state = REAGENT_STATE_LIQUID
+	color = "#C8A5DC" //rgb: 200, 165, 220
+	overdose_am = REAGENTS_OVERDOSE
+	pain_resistance = 20
+	density = 1.8
+	specheatcap = 0.66
+
+/datum/reagent/mannitol/on_mob_life(var/mob/living/M)
+
+	if(..())
+		return 1
+
+	if(M.getBrainLoss() > 10 && prob(40))
+		M.adjustBrainLoss(-1 * REM)	
```

Comment:
Make it work as a weak painkiller + brain damage inducer instead of enabling TG refugees.

Reply:
fixed